test/unit/filters/richTextFilterSpec.js

```diff
@@ -0,0 +1,32 @@
+/* global describe, it, inject, expect, beforeEach */
+
+describe('richText filter', function () {
+  var $filter, richTextFilter
```

Comment:
Today I've learned that you can share variables by assigning them to `this`.

See above for explanation.